awx/main/scheduler/task_manager_models.py

```diff
@@ -34,14 +34,14 @@ def remaining_capacity(self):
 
 
 class TaskManagerInstances:
-    def __init__(self, active_tasks, instances=None):
+    def __init__(self, active_tasks, instances=None, instance_fields=['node_type', 'capacity', 'hostname', 'enabled']):
```

Comment:
Generally we want to avoid mutable kwarg defaults, so `instance_fields=('node_type', 'capacity', 'hostname', 'enabled')` may be the easy fix for that. You don't ever seem to mutate it, so it's unlikely to actually be a problem, but it's a general practice thing. Linters should start checking for this...